build.gradle.kts

```diff
@@ -105,6 +105,11 @@ val copyCurrentDistro by task<Copy> {
     into(customInstallationDir)
     exclude("**/*kotlin*")
 
+    // TODO:pm remove SPIKE
+    from(file("gradle/gradle-api-parameter-names-SPIKE.jar")) {
+        into("lib")
+    }
+
```

Comment:
❗️This would be generated by https://github.com/gradle/gradle/commit/8d75cc1900ed0f4e3464975e25c5ccae609caf6b